### PR TITLE
StorageSync ResourceProvider : Bug Fix to change parameter name

### PR DESCRIPTION
**Pull request by ankushbindlish2**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by sergey-shandar — APPROVED** [review]

**Comment by dsgouda** [comment]
@azuresdkci Retest this please

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
@jhendrixMSFT How can we fix the issues with go build here

**Comment by jhendrixMSFT** [comment]
This is the pertinent part of the error. "FATAL: System.InvalidOperationException: codegen for preview swagger Microsoft.StorageSync/preview/2018-10-01/storagesync.json must be under a preview subdirectory"
I went ahead and fixed up the readme.go.md file.

**Comment by dsgouda** [comment]
Thanks Joel, appreciate the help! :)

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-ruby

File: specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json [hunk 1]
```diff
@@ -2595,7 +2595,7 @@
             "type": "object",
             "description": "Trigger Rollover Request.",
             "properties": {
-                "certificateData": {
+                "serverCertificate": {
```
Comment (dsgouda) [inline specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2018-07-01/storagesync.json]: @ankushbindlish2 why is a stable version of the service being changed here. This is a breaking change on the service side and we should undo this change
